app/jobs/bulkrax/schedule_relationships_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Bulkrax
+  class ScheduleRelationshipsJob < ApplicationJob
+    def perform(importer_id:)
+      importer = Importer.find(importer_id)
+      pending_num = Bulkrax::Status.where(statusable_type: "Bulkrax::Entry",
```

Comment:
@orangewolf this is what I cam up with to deal with the the statuses issue for now.

Reply:
this never runs if any of the entries Fail. that doesn't seem right to me. seems like it should run if all entries are not pending (which is not failed and not complete)

Reply:
I saw this when going over things with Kiah, then forgot to change it. It's fixed now.

Reply:
ahh I didn't realize that... so something like:

```
pending_num = importer.entries.left_outer_joins(:latest_status).where('bulkrax_statuses.status_message <> ?', %w[Complete Failed]).count
```

that should grab all entries without a complete or failed status

Reply:
or should I just use the NULL one and assume that we are only looking for `pending` statuses?

Reply:
I think the null one is probably safe